src/components/Feed/Feed.js

```diff
@@ -1,13 +1,32 @@
 import React, { Fragment } from 'react'
 import moment from 'moment'
+import InsightsFeatured from '../Insight/InsightsFeatured'
 import styles from './Feed.module.scss'
 
+const FeaturedInsightsBlock = () => (
+  <section className={styles.featuredInsights}>
+    <h4 className={styles.featuredInsights__title}>Featured insights</h4>
+    <div className={styles.featuredInsights__wrapper}>
+      <div className={styles.featuredInsights__scrollableWrapper}>
+        <div className={styles.featuredInsights__scrollable}>
+          <InsightsFeatured
+            multilineTextId='InsightsBetweenFeeds'
+            maxLines={3}
+            className={styles.featuredInsights__card}
+          />
+        </div>
+      </div>
+    </div>
+  </section>
+)
+
 const Feed = ({ component: El, data, dateKey }) => {
   let lastDateKey
   return data.map((item, index) => {
     const id = item.id || index
     const date = moment(item[dateKey]).format('MMM D')
     const isNotSameAsLastDate = date !== lastDateKey
+    const showAfterElNumber = 2
```

Comment:
This variable should be defined statically and named as the constant. Currently, it's created on every `Feed` rerender